sporks.json

```diff
@@ -1,950 +1,995 @@
-    {
-      "spork-id": "Devnet19",
-      "spork-date": "2021-02-03T16:00:00Z",
-      "root-height": "16483518",
-      "root-parent-id": "4220911048404bd3a7733ab6219531a5945dc20020f86869bcd6422c3a6e3f76",
-      "root-state-commit": "f6eea9b652a7df433b80d8d647ba414aabcfa03153dd1cba33048c27ce888097",
-      "git-commit-hash": "eb11ae095df6db6e856b1a8e824f03ce4c713b19",
-      "state-artefacts" : {
-        "gcp": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        },
-        "s3": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        }
-      },
-      "flow-go-tag": "v0.14.0",
-      "flow-dps-tag": "",
-      "cadence-tag": "",
-      "docker-tag": "v0.14.0",
-      "access-node": ""
-    },
-    {
-      "spork-id": "Devnet18",
-      "spork-date": "2020-12-11T16:00:00Z",
-      "root-height": "17756122",
-      "root-parent-id": "52ace0d0c8d4e574213fe98e19a5043f215bee992659bd1ef35c5758acf54d1b",
-      "root-state-commit": "b1412c9d453b1c10d3ace6111e00ac804d564ba3d3295002bedb077685c7da73",
-      "git-commit-hash": "115ccee9cae920ee5bc309537cf43a9b152a1cf9",
-      "state-artefacts" : {
-        "gcp": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        },
-        "s3": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        }
-      },
-      "flow-go-tag": "v0.13.0",
-      "flow-dps-tag": "",
-      "cadence-tag": "",
-      "docker-tag": "v0.13.0",
-      "access-node": ""
-    },
-    {
-      "spork-id": "Devnet17",
-      "spork-date": "2020-11-27T16:00:00Z",
-      "root-height": "16483518",
-      "root-parent-id": "96d452f9d2a15fafab720f1809fc490b019b23e42c885590e62085e48b8c2b6b",
-      "root-state-commit": "0b45f6055a1f09b413d098997cc0e7c9a0ef19eac8b4d294085b0e2f436c6bff",
-      "git-commit-hash": "ef54713595d0fe1e0bb4b14e9469c4a64dfaf6e7",
-      "state-artefacts" : {
-        "gcp": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        },
-        "s3": {
-          "root-checkpoint-file": "",
-          "root-protocol-state-snapshot": "",
-          "node-info": "",
-          "execution-state-wal-chunks": ""
-        }
-      },
-      "flow-go-tag": "v0.12.1",
-      "flow-dps-tag": "",
-      "cadence-tag": "",
-      "docker-tag": "v0.12.1",
-      "access-node": ""
+  "networks" : [
+    {
+      "name": "mainnet",
+      "sporks": [
+        {
+          "id": 14,
+          "name": "Mainnet-14",
+          "spork-date": "2021-10-06T16:00:00Z",
```

Comment:
this includes the time, so maybe spork-time?

Reply:
Yeah this is the spork time, but adjusted for UTC given the time format

Reply:
I meant the naming of the field

Reply:
Ah, fair point. Will do